eventuals/transformer.h

```diff
@@ -251,6 +269,16 @@ struct _TransformerFromTo final {
 
       using E = decltype(f());
 
+      static_assert(
+          HasValueFrom<E>::value,
+          "'Transformer' expects a callable that returns an eventual");
+
+      using ErrorsFromE = typename E::template ErrorsFrom<From_, std::tuple<>>;
+
+      static_assert(
+          eventuals::tuple_types_subset_v<ErrorsFromE, Errors_>,
```

Comment:
```suggestion
          eventuals::tuple_types_subset_subtype_v<ErrorsFromE, Errors_>,
```

Please DOUBLE CHECK THIS EVERYWHERE!!!